Test failure: `com.linecorp.armeria.xds.client.endpoint.RampingUpTest.checkEndpointsAreRampedUp()`

```
java.lang.AssertionError: 
Expecting actual:
  [null,
    Endpoint{a.com:80, weight=1, attributes=[com.linecorp.armeria.xds.client.endpoint.XdsAttributeKeys#LOCALITY_LB_ENDPOINTS_KEY=locality {
}
lb_endpoints {
  endpoint {
    address {
      socket_address {
        address: "a.com"
        port_value: 80
      }
    }
  }
  health_status: HEALTHY
  metadata {
  }
  load_balancing_weight {
    value: 10
  }
}
lb_endpoints {
  endpoint {
    address {
      socket_address {
        address: "b.com"
        port_value: 80
      }
    }
  }
  health_status: HEALTHY
  metadata {
  }
  load_balancing_weight {
    value: 10
  }
}
, com.linecorp.armeria.internal.client.endpoint.RampingUpKeys#createdAtNanos=31975871750669554, com.linecorp.armeria.xds.client.endpoint.XdsAttributeKeys#LB_ENDPOINT_KEY=endpoint {
  address {
    socket_address {
      address: "a.com"
      port_value: 80
    }
  }
}
health_status: HEALTHY
metadata {
}
load_balancing_weight {
  value: 10
}
]},
    Endpoint{b.com:80, weight=1, attributes=[com.linecorp.armeria.xds.client.endpoint.XdsAttributeKeys#LOCALITY_LB_ENDPOINTS_KEY=locality {
}
lb_endpoints {
  endpoint {
    address {
      socket_address {
        address: "a.com"
        port_value: 80
      }
    }
  }
  health_status: HEALTHY
  metadata {
  }
  load_balancing_weight {
    value: 10
  }
}
lb_endpoints {
  endpoint {
    address {
      socket_address {
        address: "b.com"
        port_value: 80
      }
    }
  }
  health_status: HEALTHY
  metadata {
  }
  load_balancing_weight {
    value: 10
  }
}
, com.linecorp.armeria.internal.client.endpoint.RampingUpKeys#createdAtNanos=31975871750669554, com.linecorp.armeria.xds.client.endpoint.XdsAttributeKeys#LB_ENDPOINT_KEY=endpoint {
  address {
    socket_address {
      address: "b.com"
      port_value: 80
    }
  }
}
health_status: HEALTHY
metadata {
}
load_balancing_weight {
  value: 10
}
]}]
to contain exactly in any order:
  [Endpoint{a.com:80, weight=1000}, Endpoint{b.com:80, weight=1000}]
but the following elements were unexpected:
  [null]

	at com.linecorp.armeria.xds.client.endpoint.RampingUpTest.checkEndpointsAreRampedUp(RampingUpTest.java:137)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at java.util.ArrayList.forEach(ArrayList.java:1259)
	at java.util.ArrayList.forEach(ArrayList.java:1259)

```

--- a/xds/src/test/java/com/linecorp/armeria/xds/client/endpoint/RampingUpTest.java
+++ b/xds/src/test/java/com/linecorp/armeria/xds/client/endpoint/RampingUpTest.java
@@ -41,6 +41,7 @@ import com.linecorp.armeria.client.ClientRequestContext;
 import com.linecorp.armeria.client.Endpoint;
 import com.linecorp.armeria.client.WebClient;
 import com.linecorp.armeria.client.endpoint.EndpointGroup;
+import com.linecorp.armeria.common.CommonPools;
 import com.linecorp.armeria.common.HttpMethod;
 import com.linecorp.armeria.common.HttpRequest;
 import com.linecorp.armeria.common.HttpResponse;
@@ -176,8 +177,8 @@ class RampingUpTest {
     private static Set<Endpoint> selectEndpoints(int weight, EndpointGroup xdsEndpointGroup) {
         final Set<Endpoint> selectedEndpoints = new HashSet<>();
         for (int i = 0; i < weight * 2; i++) {
-            selectedEndpoints.add(xdsEndpointGroup.selectNow(ctx()));
-            selectedEndpoints.add(xdsEndpointGroup.selectNow(ctx()));
+            selectedEndpoints.add(xdsEndpointGroup.select(ctx(), CommonPools.workerGroup()).join());
+            selectedEndpoints.add(xdsEndpointGroup.select(ctx(), CommonPools.workerGroup()).join());
         }
         return selectedEndpoints;
     }

